mysql-test/r/percona_mysqldump_innodb_optimize_keys.result

```diff
@@ -7,7 +7,7 @@ CREATE TABLE t1 (a INT NOT NULL PRIMARY KEY, b INT, KEY(b)) ENGINE=MyISAM;
 /*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
 /*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
 /*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
-/*!40101 SET NAMES utf8 */;
+ SET NAMES utf8mb4 ;
```

Comment:
Does extra leading space and space before the semicolon come from our patches or upstream?

Reply:
It's upstream at https://github.com/percona/percona-server/commit/4491eaf786696c5423560747eebc75b4132d082f